code/modules/power/supermatter/supermatter.dm

```diff
@@ -28,6 +29,7 @@
 #define BZ_TRANSMIT_MODIFIER -2
 #define TRITIUM_TRANSMIT_MODIFIER 30 //We divide by 10, so this works out to 3
 #define PLUOXIUM_TRANSMIT_MODIFIER -5 //Should halve the power output
+#define FREON_TRANSMIT_MODIFIER -5 //used to almostremove the power generation
```

Comment:
comment fixed in the next commit, to be really fair, this modifier doesnt matter too much, in fact i'm thinking of removing it because the power generation stop is calculated by amount and not by this value